website/content/api-docs/secret/pki.mdx

```diff
@@ -1602,3 +1602,30 @@ $ curl \
     --data @payload.json \
     http://127.0.0.1:8200/v1/pki/tidy
 ```
+
+# Cluster Scalability
```

Comment:
Is it worth also talking about generate_lease?

Reply:
Ah, yes, good call.

Reply:
Actually, if generate_lease is set true, the perf standby will handle it.  It will do an RPC at the end to the active node to register the lease.  So it's kind of a halfway point between fully handled on the perf standby and not handled at all.

Reply:
Noted that.